crowbar_framework/app/models/deployer_service.rb

```diff
@@ -120,7 +142,9 @@ def transition(inst, name, state)
       @logger.error("Failed to allocate admin address for: #{node.name}: #{result[0]}") if result[0] != 200
       if result[0] == 200
         address = result[1]["address"]
-        boot_ip_hex  = sprintf("%08X",address.split(".").inject(0){ |acc,i|(acc << 8)+i.to_i })
+        boot_ip_hex = sprintf("%08X", address.split(".").inject(0){ |acc,i| (acc << 8) + i.to_i })
```

Comment:
Style/SpaceAfterComma: Space missing after comma. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)<br>Style/SpaceBeforeBlockBraces: Space missing to the left of {.